### PR TITLE
fix(plugin-meetings): fix broken api documentation

### PR DESCRIPTION
**Pull request by arun3528**
# COMPLETES #[SPARK-395102](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-395102) and [SPARK-391970](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-391970)

## This pull request addresses

< DESCRIBE THE CONTEXT OF THE ISSUE >

## by making the following changes

< DESCRIBE YOUR CHANGES >

### Change Type

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

< ENUMERATE TESTS PERFORMED, WHETHER MANUAL OR AUTOMATED >

### I certified that

- [ ] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [ ] I discussed changes with code owners prior to submitting this pull request

- [ ] I have not skipped any automated checks
- [ ] All existing and new tests passed
- [ ] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: patches/documentation+14.0.1.patch [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
++      ...(TYPESCRIPT_EXTS[path.extname(file || '')] || ['flow', 'jsx']),
++    ],
+   };
+ }
+ 
+@@ -50,9 +50,7 @@ function getParserOpts(file) {
+  */
+ export function commentToFlow(source) {
+   if (!/@flow/.test(source)) return source;
+-  return source
```
Comment (Neeraj-swarnkar) [inline patches/documentation+14.0.1.patch:53]: This seems just formatting if I am not wrong.

File: patches/documentation+14.0.1.patch [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
++  '.tsx': ['typescript', 'jsx'],
+ };
+ 
+ // this list is roughly the same as the one in prettier
+@@ -14,7 +14,7 @@ export const standardBabelParserPlugins = [
+   'classProperties',
+   'classStaticBlock',
+   'decimal',
+-  ['decorators', { decoratorsBeforeExport: false }],
++  'decorators-legacy',
```
Comment (Neeraj-swarnkar) [inline patches/documentation+14.0.1.patch:19]: This is for applying the patch for 14 version,

Comment (arun3528) [reply patches/documentation+14.0.1.patch:19]: ya

File: patches/documentation+14.0.1.patch (hunk 1 above)
Comment (arun3528) [reply patches/documentation+14.0.1.patch:53]: yes just formatting should not affect the PR

**Review by Neeraj-swarnkar — APPROVED** [review]
Looks Good to me.

**Pull request merged**